draft-thomson-webpush-http2.xml

```diff
@@ -109,6 +107,56 @@
         overhead costs.
       </t>
       <t>
+        A push server that does not support reliable delivery over intermittent
+        network connections or failing applications on devices, forces the
+        device to acknowledge receipt directly to the application server,
+        incurring additional power drain in order to establish (usually secure)
+        connections to the individual application servers.
```

Comment:
I don't think that this belongs here.  It might have been important to establish how your draft was different to my original draft, but I'd rather see this moved to a section on acknowledgement.